e2e_test/e2e_test.go

```diff
@@ -450,6 +453,40 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 				pingTest(dpuClientSet, dpuRestConfig, nfPod, pod1_ip, nfPod.Name, pod1.Name)
 				pingTest(dpuClientSet, dpuRestConfig, nfPod, pod2_ip, nfPod.Name, pod2.Name)
 			})
+			g.It("Should support Network-function -> external", func() {
+				if skipNetworkFunctionTesting {
+					g.Skip("Skipping Network Function Testing")
+				}
+
+				fmt.Printf("Assigning NF ingress port IP %s\n", nfIngressIp)
+				_, err := testutils.ExecInPod(dpuClientSet, dpuRestConfig, nfPod, fmt.Sprintf("ip addr add %s dev net2", nfIngressIp+"/24"))
```

Comment:
Could you explain why you need to run these ip route commands?

Reply:
You mean that the operator does not write IPAM into the config JSON, is that correct?